lambdas/process-journey-step/src/main/java/uk/gov/di/ipv/core/processjourneystep/ProcessJourneyStepHandler.java

```diff
@@ -121,6 +122,10 @@ private Map<String, Object> executeJourneyEvent(
                     journeyStep,
                     ipvSessionItem);
 
+            clearOauthSessionIfExists(ipvSessionItem);
+
+            ipvSessionService.updateIpvSession(ipvSessionItem);
```

Comment:
Might be worth moving this update call inside the `clearOauthSessionIfExists` method, so that we are only making the dynamoDB update when we need to?

Reply:
Oh wait never mind, I see now its because we are making the update no matter anyways for other reasons. Ignore the above